server/dummydatabase/dummydatabase.js

```diff
@@ -0,0 +1,22 @@
+const requests = [{
+  id: 1,
+  requestTitle: 'Faulty TV',
+  status: {
+    resolved: true,
+    approved: true,
```

Comment:
Why is status an object? Do you think its ok?

Reply:
Since they hold the same values (boolean). I believe it will be ideal if they are grouped together